pilot/pkg/config/kube/gateway/conversion.go

```diff
@@ -51,12 +52,13 @@ import (
 
 func sortConfigByCreationTime(configs []config.Config) {
 	sort.Slice(configs, func(i, j int) bool {
-		if configs[i].CreationTimestamp.Equal(configs[j].CreationTimestamp) {
-			in := configs[i].Namespace + "/" + configs[i].Name
-			jn := configs[j].Namespace + "/" + configs[j].Name
-			return in < jn
+		if r := configs[i].CreationTimestamp.Compare(configs[j].CreationTimestamp); r != 0 {
+			return r == -1 // -1 means i is less than j, so return true
 		}
-		return configs[i].CreationTimestamp.Before(configs[j].CreationTimestamp)
+		if r := cmp.Compare(configs[i].Namespace, configs[j].Namespace); r != 0 {
```

Comment:
how about cmp compared with < operator?

Reply:
There is basically no difference in performance, just to keep the same pattern.

https://github.com/istio/istio/blob/27a44759d96d7e532a62321c8a2e2f33f6920e1f/pkg/slices/slices_test.go#L820